lib/p2p/packets/types/SwapRequestPacket.ts

```diff
@@ -11,14 +11,18 @@ export type SwapRequestPacketBody = {
 };
 
 class SwapRequestPacket extends Packet<SwapRequestPacketBody> {
-  public get type() {
+  public get type(): PacketType {
```

Comment:
Fwiw the restriction to return a `PacketType` already exists because it's declared on the base `Packet` class, so it's not necessary to explicitly state it here for the compiler to enforce it. But if you think it's more readable this way it's fine to leave these changes in place that's fine.

Reply:
It was necessary to be declared for `responseType`, so I preferred the style to be unified.